pom.xml

```
@@ -108,7 +108,19 @@
 
 		<dependency>
 			<groupId>org.seleniumhq.selenium</groupId>
-			<artifactId>selenium-java</artifactId>
+			<artifactId>selenium-api</artifactId>
+			<version>3.141.59</version>
```

If the selenium versions always have to be the same for all three dependencies, you could use something like
```suggestion
			<version>${selenium.version}</version>
```
and then define a property `<selenium.version>3.141.59</selenium.version>` in the `properties` section.